docs/sematic-vs.md

```diff
@@ -56,35 +56,97 @@ But bear with us, we'll catch up fast!
 
 {% endhint %}
 
-### Airflow
+### ... Airflow
 
 Airflow is probably the most popular and widespread way to build end-to-end
-pipelines, since it was the first to provide such abilities.
+pipelines, since it was one of the first to provide such abilities.
 
 Sematic differs from Airflow in the following ways:
 
-* **Iterative development** – change code, run in the cloud, visualize, repeat. In Airflow, you can either run a pipeline on a local Airflow instance, or if you want to run it in the cloud, you must merge your code and deploy it to the cloud instance which adds many steps to your iteration workflow.
+* **Iterative development** – change code, run in the cloud, visualize, repeat. In Airflow, you can either run a pipeline on a local Airflow instance, or if you want to run it in the cloud, you must merge your code and deploy it to the cloud instance which adds many steps to your iteration workflow. In Sematic, you can execute your pipeline entirely locally while you develop: you can even run it
+without running a local server if you want to!
 
 * **Semantic UI** – Sematic brings visualizations for your functions inputs and outputs straight to the forefront. No need to take care of persisting things elsewhere or fetching them into a local notebook.
 
 * **Dynamic graph** – Sematic lets up loop over configurations, run different branches of your pipelines depending on the outcome of upstream steps, and even do hyperparameter tuning.
 
-### Kubeflow Pipelines
+* **Dead simple I/O between steps** – Sematic makes I/O between steps in your pipelines as simple as passing an output of one
+python function as the input of another. Airflow provides APIs like
+[XComs](https://airflow.apache.org/docs/apache-airflow/stable/concepts/xcoms.html) which can pass data between tasks, but involves
+some boilerplate around explicitly pushing/pulling data around, and coupling producers and consumers via named data keys.
 
-* **Barrier to entry** – Using KFP requires quite a bit of knowledge about
+### ... Kubeflow Pipelines
+
+* **Low Barrier to entry** – Using KFP requires quite a bit of knowledge about
   Kubernetes, Docker images, Argo, etc. These things take a lot of time to
   master. Sematic tries to provide working solutions out-of-the-box so that you
   can focus on your expertise instead of infrastructure.
 
-* **Dynamic graph** – In KFP, DAGs are fixed ahead of time. Sematic lets up loop over configurations, run different branches of your pipelines depending on the outcome of upstream steps, and even do hyperparameter tuning.
+* **Dynamic graph** – In KFP, DAGs are fixed ahead of time. While it does provide
+[some mechanisms](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.dsl.html#kfp.dsl.Condition)
```

Comment:
Same here re: links. "some mechanisms" is sufficient.

Reply:
changed!